### PR TITLE
Fix notifier role defaults never working

### PR DESCRIPTION
**Pull request by maxrothman**
Refactored `NOTIFIER_USE_GIT_IDENTITY` to be represented by `NOTIFIER_GIT_IDENTITY` being an empty string.

### CODE REVIEW

File: playbooks/roles/notifier/tasks/deploy.yml [hunk 1]
```diff
@@ -16,27 +16,19 @@
...
-  when: NOTIFIER_USE_GIT_IDENTITY
+  when: NOTIFIER_GIT_IDENTITY != ""
 
 - name: install read-only ssh key
   copy: >
     content="{{ NOTIFIER_GIT_IDENTITY }}" dest={{ notifier_git_identity }}
     force=yes owner={{ NOTIFIER_USER }} mode=0600
-  when: NOTIFIER_USE_GIT_IDENTITY
-
-- name: checkout theme
```
Comment (fredsmith) [inline playbooks/roles/notifier/tasks/deploy.yml]: We need to leave theming in.  This shouldn't be removed.  Perhaps NOTIFIER_THEME_NAME just needs a default?

Comment (maxrothman) [inline playbooks/roles/notifier/tasks/deploy.yml]: Whoops, I don't think I meant to remove all of that. I'll fix.

**Comment by fredsmith** [comment]
:+1:

**Comment by e0d** [comment]
:+1:

**Pull request merged**